docs/core/runtime-config/envvars.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Runtime configuration with environment variables"
+description: "Use environment variables to configure the runtime behavior of .NET Core applications."
+ms.date: "06/19/2019"
+author: "rpetrusha"
+ms.author: "ronpet"
+---
+# Runtime configuration with environment variables
+
+.NET Core supports a number of environment variables that you can use to configure your applications. 
+
+The following sections list selected supported environment variables by category. For a complete list of supported, as well as unsupported and internal environment variables, see [Host Configuration Knobs](https://github.com/dotnet/coreclr/blob/a28b25aacdcd2adb0fdfa70bd869f53ba6565976/Documentation/project-docs/clr-configuration-knobs.md).
+
+> [!NOTE]
+> This documentation is a work in progress. If you notice that the information presented here is either incomplete or inaccurate, either [open an issue](https://github.com/dotnet/docs/issues) to let us know about it, or [submit a pull request](https://github.com/dotnet/docs/pulls) to add or correct the information. For information on submitting pull requests for the dotnet/docs repo, see the [contributor's guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md).
+
+## Application domains
+
+|Variable name|Variable value|Description|
+|--|--|--|
+|`COMPlus_ADULazyMemoryRelease`|0 or 1|Determines whether a lazy memory release occurs when an app domain is unloaded. It is on (1) by default; it should be turned off (0) to catch memory leaks, in which case the unload should be immediately followed by a garbage collection.|
+|`COMPlus_ADURetryCount`|*n*|Controls the timeout of an app domain unload. It is used for workarounds when a machine is too slow, there are network issues, etc.|
+|`COMPlus_FinalizeOnShutdown`|0 or 1|When enabled, on shutdown, it blocks all user threads and calls finalizers for all finalizable objects, including live objects.|
+
+## Assembly loading
+
+|Variable name|Variable value|Description|
```

Comment:
This is a very special setting for Visual Studio designer and nobody else. We should not be documenting it.